INSTALL.md

```diff
@@ -36,6 +36,22 @@ make bash
 
 All the source and dependencies for this project will be in place, making it possible to run porta and the tests from inside the container. See [Run Porta](#run-porta)
 
+#### Running the application
+
+If you want to run the application using docker, you can use the docker compose file to start all dependencies.
+
+In the first time, you need to setup the database. To configure it, run: (the Master and User password will be `p`)
+
+```
+MASTER_PASSWORD=p USER_PASSWORD=p make dev-setup
+```
+
+and to start the application and all dependencies:
+
+```
+make up
+```
+
```

Comment:
Once it's running, how do you open the app?

Reply:
Like locally. For example, I have a dnsmasq rule with example.com and I access this way:

http://master-account.example.com.lvh.me:3000

Reply:
Lvh.me or nip.io should be documented for the sake of new users in the community
I think it is already

Reply:
About this instruction, can’t all the targets be chained in one and only target? `make all` which can be the default one